pkg/domain/infra/abi/play.go

```diff
@@ -313,6 +334,68 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 	return &report, nil
 }
 
+// playKubePVC creates a podman volume from a kube persistent volume claim.
+func (ic *ContainerEngine) playKubePVC(ctx context.Context, pvcYAML *v1.PersistentVolumeClaim, options entities.PlayKubeOptions) (*entities.PlayKubeReport, error) {
+	var report entities.PlayKubeReport
+	opts := make(map[string]string)
+
+	// Get pvc name.
+	// This is the only required pvc attribute to create a podman volume.
+	name := pvcYAML.GetName()
+	if strings.TrimSpace(name) == "" {
+		return nil, fmt.Errorf("persistent volume claim name can not be empty")
+	}
+
+	// Create podman volume options.
+	volOptions := []libpod.VolumeCreateOption{
+		libpod.WithVolumeName(name),
+		libpod.WithVolumeLabels(pvcYAML.GetLabels()),
+	}
+
+	// Get pvc annotations and create remaining podman volume options if available.
+	// These are podman volume options that do not match any of the persistent volume claim
+	// attributes, so they can be configured using annotations since they will not affect k8s.
+	for k, v := range pvcYAML.GetAnnotations() {
+		switch k {
+		case "podman.volume/driver":
```

Comment:
2 questions for consideration:
- should these be kept as constants anywhere?
- often, these special annotations follow the format of a url: `io.podman.volume` does it make sense to apply that here?

Reply:
1. Let me change that.
2. You're correct. The url format does not provide any special functionality but it is better to stick to the k8s syntax. What about something like this `volume.podman.io/driver` based on `node.kubernetes.io/instance-type`?

Reply:
This has been changed.